### PR TITLE
Added IIS Administration API sample dockerfile.

### PR DESCRIPTION
**Pull request by jimmyca15**
The sample for the IIS Administration API comes with a README containing instructions of how to setup the files for building a container image.

### CODE REVIEW

File: samples/AdministrationApi/dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# escape=`
+
+FROM microsoft/iis
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+#
+# Download and install IIS Administration API
+RUN `
```
Comment (shirhatti) [inline samples/AdministrationApi/dockerfile]: Nit: delete exe

File: samples/AdministrationApi/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Adding An Api Key
+
+Before building this dockerfile, the _api-keys.json_ file should be updated with a valid api key. To acquire an api key use the _Generate-ApiKey.ps1_ PowerShell script found at https://github.com/Microsoft/IIS.Administration/tree/dev/scripts/utils. This script will generate an access token plus a token hash. The access token is a **secret** used to communicate with the API and should not be shared with anyone. Uncomment the example api key in _api-keys.json_ and update the token_hash property with the token hash value generated by the script.
+
+
+First generate an access token. The token values in this example have been removed.
+
+```
+PS C:\temp> .\Generate-ApiKey.ps1
```
Comment (shirhatti) [inline samples/AdministrationApi/README.md]: I'd rather something like this I don't have to clone to repo. 
```powershell
powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('htt
ps://raw.githubusercontent.com/Microsoft/IIS.Administration/dev/scripts/utils/Generate-ApiKey.ps1'))}"
```

Comment (jimmyca15) [reply samples/AdministrationApi/README.md]: @shirhatti 

Great suggestion!

**Comment by jimmyca15** [comment]
c:\src\repos\iisadmin>docker history --human 647bda3a2def
IMAGE               CREATED              CREATED BY                                      SIZE                COMMENT
647bda3a2def        33 seconds ago       powershell -Command $ErrorActionPreference =…   41kB
c1b87eef85d4        37 seconds ago       powershell -Command $ErrorActionPreference =…   38.7MB
d7663b49a4a2        About a minute ago   powershell -Command $ErrorActionPreference =…   42.3kB
063e9402bb14        About a minute ago   powershell -Command $ErrorActionPreference =…   301MB
ff5feb4732df        11 days ago          powershell -Command $ErrorActionPreference =…   41kB
6608c8e5b344        2 weeks ago          cmd /S /C #(nop)  ENTRYPOINT ["C:\\ServiceMo…   41kB
\<missing\>           2 weeks ago          cmd /S /C #(nop)  EXPOSE 80                     41kB
\<missing\>           2 weeks ago          cmd /S /C #(nop) ADD file:693ab629b720d02049…   168kB
\<missing\>           2 weeks ago          cmd /S /C powershell -Command Add-WindowsFea…   301MB
\<missing\>           2 weeks ago          Install update 10.0.14393.2007                  2.76GB
\<missing\>           13 months ago        Apply image 10.0.14393.0                        7.68GB

@shirhatti 

From the docker file it looks like the two powershell steps could be combined into one, but I put the copy in between so changing out the configuration files wouldn't require a rebuild of the installation layer.

**Comment by shirhatti** [comment]
@jimmyca15 Sorry I forgot to approve this. You can go ahead and merge

**Pull request merged**